src/urllib3/response.py

```diff
@@ -852,7 +887,7 @@ def read_chunked(self, amt=None, decode_content=None):
                     yield decoded
 
             # Chunk content ends with \r\n: discard it.
-            while True:
+            while True and self._fp is not None:
```

Comment:
We can reduce this to `while self._fp is not None:`?

Reply:
done!